### ISSUE TITLE
[MU4 Issue] [Accessibility] Screen Reader doesn't read checkbox all states during changing (checked/unchecked)

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#9497]
**To Reproduce**

https://user-images.githubusercontent.com/90187801/137898361-31497955-14d2-41c4-ac57-d6b86147e4ce.mp4

**Expected behavior**
Screen Reader should read all two states (now it reads only the first state)

**Desktop (please complete the following information):**
 **Occurs** with Narrator/NVDA (Win10),  Orca (Linux Mint 20.2)
**Doesn't occur** with VoiceOver on Mac11

**Comment by DmitryArefiev** [comment]
Tested with Narrator/NVDA (Win10), Orca (Linux Mint 20.2) and VoiceOver (Mac11) - DONE
